posthog/tasks/email.py

```diff
@@ -425,6 +426,26 @@ def send_two_factor_auth_backup_code_used_email(user_id: int) -> None:
     message.send(send_async=False)
 
 
+@shared_task(**EMAIL_TASK_KWARGS)
+def send_revenue_analytics_sync_complete_email(team_id: int, source_id: int) -> None:
+    team: Team = Team.objects.get(pk=team_id)
+    source: ExternalDataSource = ExternalDataSource.objects.get(team=team, pk=source_id)
+
+    message = EmailMessage(
+        use_http=True,
+        campaign_key=f"revenue_analytics_sync_complete_{team_id}_{source_id}",
+        template_name="revenue_analytics_sync_complete",
+        subject=f"Revenue analytics sync complete for {source.source_type}",
+        template_context={
+            "source_type": source.source_type,
+        },
+    )
+
+    for membership in get_members_to_notify(team):
+        message.add_recipient(email=membership.user.email, name=membership.user.first_name)
+    message.send(send_async=False)
```

Comment:
Maybe I should just use `send_message_to_all_staff_users` instead? Probably easier than adding all of these permissions, right?